frontend/public/index.html

```diff
@@ -6,19 +6,28 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     [[ if eq .Branding "origin" ]]
-    <title>OpenShift Origin</title>
+    <title>OKD</title>
+    <link rel="shortcut icon" href="<%= require('./imgs/okd/favicon.png') %>">
+    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="<%= require('./imgs/okd/apple-touch-icon-precomposed.png') %>">
+    <link rel="mask-icon" href="<%= require('./imgs/okd/mask-icon.svg') %>" color="#DB242F">
+    <meta name="application-name" content="OKD">
+    <meta name="msapplication-TileColor" content="#000000">
+    <meta name="msapplication-TileImage" content="<%= require('./imgs/okd/mstile-144x144.png') %>">
     [[ end ]]
     [[ if eq .Branding "ocp" ]]
     <title>OpenShift Container Platform</title>
+    <link rel="shortcut icon" href="<%= require('./imgs/ocp/favicon.png') %>">
+    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="<%= require('./imgs/ocp/apple-touch-icon-precomposed.png') %>">
+    <link rel="mask-icon" href="<%= require('./imgs/ocp/mask-icon.svg') %>" color="#DB242F">
+    <meta name="application-name" content="OpenShift Container Platform">
+    <meta name="msapplication-TileColor" content="#000000">
+    <meta name="msapplication-TileImage" content="<%= require('./imgs/ocp/mstile-144x144.png') %>">
```

Comment:
You'll need these same icons for Online branding, too. Pull the icon into a separate if/else

```html
[[ if eq .Branding "origin" ]]
    <link rel="shortcut icon" href="<%= require('./imgs/okd/favicon.png') %>">
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="<%= require('./imgs/okd/apple-touch-icon-precomposed.png') %>">
    <link rel="mask-icon" href="<%= require('./imgs/okd/mask-icon.svg') %>" color="#DB242F">
    <meta name="application-name" content="OKD">
    <meta name="msapplication-TileColor" content="#000000">
    <meta name="msapplication-TileImage" content="<%= require('./imgs/okd/mstile-144x144.png') %>">
[[ else ]]
    <link rel="shortcut icon" href="<%= require('./imgs/ocp/favicon.png') %>">
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="<%= require('./imgs/ocp/apple-touch-icon-precomposed.png') %>">
    <link rel="mask-icon" href="<%= require('./imgs/ocp/mask-icon.svg') %>" color="#DB242F">
    <meta name="application-name" content="OpenShift Container Platform">
    <meta name="msapplication-TileColor" content="#000000">
    <meta name="msapplication-TileImage" content="<%= require('./imgs/ocp/mstile-144x144.png') %>">
[[ end ]]
```

Reply:
`application-name` should stay with the title, though